amgut/handlers/change_pass_verify.py

```diff
@@ -53,9 +53,9 @@ def reset_pass_and_email(self, new_password, confirm_password, email,
             self.render('change_pass_verify.html', email='', kitid='',
                         passocde='', new_password='',
                         confirm_password='', result=4, message='',
```

Comment:
Old code that shows different messages depending on number. Outside of this PR scope to change.